configs/config/pretrain/byol/byol_8node_resnet.yaml

```diff
@@ -32,7 +32,7 @@ config:
           p: 0.5
         - name: ImgPilColorDistortion
           strength: 0.5
-          saturation: 0.4
+          #saturation: 0.4
```

Comment:
Revert this comment.

Reply:
Wrt (https://github.com/pranavsinghps1/vissl/blob/53bf3fc55b70c23d7fae6e59020f69b39f738ac0/vissl/data/ssl_transforms/img_pil_color_distortion.py#L24) we are passing 0.5 as strength and Saturation by default has 0.8 value, and sol the value would become 0.5*0.8=0.4, so passing the saturation variable would make it redundant, or shall we still pass it just to be sure?

Reply:
We need saturation = 0.2 As per Table 6.

Passing in saturation as 0.4, will get saturation to 0.2 since we multiply it by strength=0.5

Reply:
Understood!

Reply:
Indeed Double checked the values, Uncommented saturation.